src/probnum/linalg/solvers/information_ops/_linear_solver_information_op.py

```diff
@@ -9,12 +9,14 @@
 class LinearSolverInformationOp(abc.ABC):
     r"""Information operator of a (probabilistic) linear solver.
 
-    For a given action, the information operator collects information about the linear system to be solved.
+    For a given action, the information operator collects information about the
+    linear system to be solved.
 
     See Also
     --------
     MatVecInformationOp: Collect information via matrix-vector multiplication.
-    ProjectedResidualInformationOp: Collect information via a projection of the current residual.
+    ProjectedResidualInformationOp: Collect information via a projection of the
+    current residual.
```

Comment:
```suggestion
    ProjectedResidualInformationOp: Collect information via a projection of the current residual.
```
Similar as above: current residual will be put into a new line in the compiled docs which is undesirable.